mantle/storage/bucket.go

```diff
@@ -50,7 +51,8 @@ type Bucket struct {
 }
 
 func NewBucket(client *http.Client, bucketURL string) (*Bucket, error) {
-	service, err := storage.New(client)
+	ctx := context.Background()
```

Comment:
It would be good to receive this `ctx` from the caller instead, and wire it through consumers.